src/StockportWebapp/Views/stockportgov/Shared/Semantic/Showcase/SocialMediaLinks.cshtml

```diff
@@ -3,12 +3,12 @@
 <section>
     <h2>@Model.SocialMediaLinksSubheading</h2>
     <div class="showcase-button-container">
-    @foreach (var link in Model.SocialMediaLinks)
-    {
-        <a as-link="true" class="showcase-button" href="@link.Url">
-            <span class="@link.Icon fa 2x" aria-hidden="true"></span>
-            <span>@link.Title</span>
-        </a>
-    }
+        @foreach (var link in Model.SocialMediaLinks)
+        {
+            <a as-link="true" class="showcase-button" href="@link.Url">
+                <span class="@link.Icon fa 2x" aria-label="@link.Platform @link.Title"></span>
```

Comment:
The properties need to be the other way round. Title first then account name not platform